tests/data/riemannian_metric_data.py

```diff
@@ -129,6 +129,67 @@ def inner_product_test_data(self):
         ]
         return self.generate_tests(smoke_data, random_data)
 
+    def normalize_test_data(self):
+        n_points = 10
+        point_single = self.euc.random_point()
+        vector_single = self.euc.random_point()
+        point_multiple = self.euc.random_point(n_points)
+        vector_multiple = self.euc.random_point(n_points)
+        random_data = [
```

Comment:
Maybe `single_point`, `single_vector`, `multiple_points`, `multiple_vectors` would be more natural ?